pkg/tfgen/installation_docs.go

```diff
@@ -91,3 +95,35 @@ func writeInstallationInstructions(goImportBasePath, providerName string) string
 		goImportBasePath,
 	)
 }
+
+func applyEditRules(contentBytes []byte, docFile *DocFile) ([]byte, error) {
+	// Obtain default edit rules for documentation files
+	edits := defaultEditRules()
+
+	// Additional edit rules for installation files
+	installationFileEdits := editRules{
+		// Replace all "T/terraform" with "P/pulumi"
+		reReplace(`Terraform`, `Pulumi`),
+		reReplace(`terraform`, `pulumi`),
+		// Replace all "H/hashicorp" strings
+		reReplace(`Hashicorp`, `Pulumi`),
+		reReplace(`hashicorp`, `pulumi`),
+		// Reformat certain headers
+		reReplace(`The following arguments are supported`,
+			`The following configuration inputs are supported`),
+		reReplace(`Argument Reference`,
+			`Configuration Reference`),
+		reReplace(`block contains the following arguments`,
+			`input has the following nested fields`),
+	}
+
+	edits = append(edits, installationFileEdits...)
```

Comment:
I think the shorter version reads better.
```suggestion
	edits := append(defaultEditRules(),
		// Replace all "T/terraform" with "P/pulumi"
		reReplace(`Terraform`, `Pulumi`),
		reReplace(`terraform`, `pulumi`),
		// Replace all "H/hashicorp" strings
		reReplace(`Hashicorp`, `Pulumi`),
		reReplace(`hashicorp`, `pulumi`),
		// Reformat certain headers
		reReplace(`The following arguments are supported`,
			`The following configuration inputs are supported`),
		reReplace(`Argument Reference`,
			`Configuration Reference`),
		reReplace(`block contains the following arguments`,
			`input has the following nested fields`),
	)
```

Reply:
works for me!